caikit_nlp/modules/text_generation/peft_prompt_tuning.py

```diff
@@ -356,18 +355,47 @@ def train(
             "<NLP46653367E>", len(train_stream) > 0, "train_stream cannot be empty"
         )
 
+        if accumulate_steps:
```

Comment:
the experiments only capture train time. accumulation steps has known to influence accuracy (we have experiments proving it ) . there is a tradeoff between train time and accuracy. I agree larger values of 16 can drastically slow down training. we are leaning towards a default of 4 as it gives best accuracy. I can re-run experiments with acc=4 to measure train time.

in any case, how about instead of deprecating hyperparameters in library, we make a guideline that informs what users pick and give them the choice to tune it based on the guideline?